tests/sql/out6.fsm

```diff
@@ -1,11 +1,12 @@
-0  ->  1 "a"; # e.g. "a"
-0  ->  2 "e"; # e.g. "e"
-1  ->  4 "b"; # e.g. "ab"
-2  ->  3 "f"; # e.g. "ef"
-3  ->  5 "\n"; # e.g. "ef\n"
-3  ->  2 "e"; # e.g. "e"
-4  ->  1 "a"; # e.g. "a"
-4  ->  2 "e"; # e.g. "e"
+
+0 -> 1 'a';
+1 -> 2 'b';
+0 -> 2;
+2 -> 0;
+
+2 -> 3 'e';
+3 -> 3 'e';
```

Comment:
Shouldn't it match the `f` in state 3, and then have a final state 4 that also epsilons back to 2?

The test NFA looks like it'd match `(ab)*e+`.